content/docs/command-reference/plots/modify.md

```diff
@@ -0,0 +1,124 @@
+# plots modify
+
+Modify [plot metrics](/doc/command-reference/plots) by specifying and saving
+plotting options.
+
+## Synopsis
+
+```usage
+usage: dvc plots modify [-h] [-q | -v] [-t [TEMPLATE]] [-x X] [-y Y]
+                        [--no-csv-header] [--title TITLE] [--xlab XLAB]
+                        [--ylab YLAB] [--unset [UNSET [UNSET ...]]]
```

Comment:
I don't think "to unset" is a real verb 😋 "unset" is an adjective meaning something is not set, but not a proper word for an action. I'd recommend "remove" or "forget".

Reply:
Hmmm. I guess this is a pervasive problem throughout several commands already, OK I'll move this to it's own issue... ⏳

Reply:
iterative/dvc/issues/3950